lib/config/presets/internal/packages.ts

```diff
@@ -135,7 +135,7 @@ export const presets: Record<string, Preset> = {
   },
   react: {
     description: 'All React packages.',
-    matchPackageNames: ['@types/react', 'react**'],
+    matchPackageNames: ['@types/react**', 'react**'],
```

Comment:
this probably lead to false positives when a package starts with `react`. this rule is not limited to npm datasource